DependencyInjection/Configuration.php

```diff
@@ -138,7 +138,8 @@ protected function configureClients(ArrayNodeDefinition $root)
                 ->validate()
                     ->ifTrue(function ($clients) {
                         foreach ($clients as $name => $config) {
-                            return $config['flexible_client'] && $config['http_methods_client'];
+                            // Make sure we only allow one of these to be true
+                            return (bool) $config['flexible_client'] + (bool) $config['http_methods_client'] + (bool) $config['batch_client'] >= 2;
```

Comment:
You can use all of them the same time. That is why I limit them. (As we did before). This could be changed when we are doing behaviors (#91)